### PR TITLE
Hello Microverse Html

### PR DESCRIPTION
**Pull request by MarkYegon7**
In the project I:

- Added linters.
- Applied principles of Github flow

### CODE REVIEW

**Review by topeogunleye — CHANGES_REQUESTED** [review]
Hi @MarkYegon7, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Try-Again

> ## (Highlights) Good Points: 💪 
> * Good commit messages. :heavy_check_mark:
> * No linter errors. :heavy_check_mark:
> * Followed Github flow :heavy_check_mark:

## Required Changes ♻️
In the case of README, kindly make use the  [Microverse README](https://github.com/microverseinc/readme-template) and also remember to personalize the template to have your project info as seen in the example  of a professional README template below, as it's a project requirement. 
  - Kindly add all sections from the Microverse README.
  - The only optional section from the README file is faq.

| professional Readme  | Your Readme that is not professional |
| ------------- | ------------- |
|  Recording 2023-02-14 at 12 52 20 (5)	  | README md - Microverse-Hello-World  WSL_ Ubuntu  - Visual Studio Code 2023-02-15 11-13-07 |

- [ ] Please kindly add a descriptive PR title and description as seen in the demo below:

pull_request_desc

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -1 +1,148 @@
...
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+
+# 📖 Microverse Hello World <a name="about-project"></a>
+
+> Describe your project in 1 or 2 sentences.
+
```
Comment (souad988) [inline README.md:26]: [ OPTIONAL ] kindly remove all default text that comes under each section as it added to guide you but should be removed after customizing each section.

**Review by souad988 — CHANGES_REQUESTED** [review]
Hi @MarkYegon7 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
✔Linters setup correctly and pass all checks. 

 ## Required Changes ♻️
_one last think before approving your work, please update your PR title to something that describes better this feature exp: Setup linters/ Add hello world html 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (TadesseDev) [inline README.md:26]: - [Optional] Please consider removing non-important contents from your readme, The template is designed by Microverse for us all to use to document our projects professionally and has it filled with content related to each project we may be working on. currently, I can see you have not removed the guiding contents  which start with `>` and `directions set by Microverser`

**Review by TadesseDev — APPROVED** [review]
### Hi @MarkYegon7 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy (1)

## Highlights
 - [ ] Good job making the changes requested by the previous code reviewer. ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**